src/main/java/ledger/user_interface/ui_controllers/window/ImportTransactionsPopupController.java

```diff
@@ -100,24 +112,42 @@ private void handleReturn(ImportController.ImportFailures importFailures) {
         for (Transaction fail : importFailures.failedTransactions) {
             // Todo: Do we even want to show the user?
         }
+        genericTransactionsWindow(importFailures.ignoredTransactions,
```

Comment:
Do we want to be showing users ignored transactions. They had to set the ignore, and the point of the ignore is to ignore.